dev-docs/source/Standards/ReleaseNotesGuide.rst

```diff
@@ -71,14 +103,40 @@ Once all the directories are in place you can add your release note as a separat
 
 You can add sub-headings to a sub-heading if you would like (e.g. ``/Workbench/Algorithm/Fitting``) that is fine to do, so long as your new branch ends with the ``New_features`` and ``Bugfixes`` folders.
 
+Adding images to release notes
```

Comment:
Have approved, but just double check this heading is appearing correctly as for some reason isn't being included on the rich diff?

Reply:
It's very strange that its not on the rich diff (it's coming up with the heading 'release notes' there rather than the correct one). Given it builds locally I'm minded to say it's probably a git glitch. However to be on the safeside perhaps whoever is the @mantidproject/gatekeepers for this could do a local build too to check the headings are as expected?

Reply:
I've built it and also confirm the heading is there in the final copy. I guess the rich diff must be confused... 🤷🏻